src/helpers/cloneIterable.js

```diff
@@ -0,0 +1,34 @@
+/** @license ISC License (c) copyright 2018 original and current authors */
+/** @author Henrique Limas (HenriqueLimas) */
+
+const isArray = require('../core/isArray')
+const isIterable = require('../core/isIterable')
+const isObject = require('../core/isObject')
+
+const err = 'cloneIterable: Iterable required'
+
+function cloneIterable(source) {
+  if (!isIterable(source)) {
+    throw new TypeError(err)
+  }
+
+  if (isArray(source)) {
+    return [].concat(source)
```

Comment:
🤔 But is not a clone, should we care if is it not a clone?

Reply:
I am thinking that because this is a core function, and we have implementations for handling stuff over arrays on the existing functions. The Array Stuff should be removed from here.

thoughts?

Reply:
We could yes, I was just thinking that considering an array as interable, we do not need to check it on the functions anymore and use always as an array. If we decide to not checking array here, probably I will remove the check for the strings too. `!isObject`

Reply:
I think the main argument is that the functions already result in new values

Reply:
I see, so let's do in this way, I will remove the array and string checks for now and see. If we think we are going to need that here, we will put back. Agree with it? @evilsoft @dalefrancis88